galite-demo/galite-vaadin/src/main/kotlin/org/kopi/galite/demo/database/Migration.kt

```diff
@@ -294,6 +292,7 @@ fun addSale(client: Int, product: Int, qty: Int) {
     it[idClt] = client
     it[idPdt] = product
     it[quantity] = qty
+    it[color] = Color((Int.MIN_VALUE..Int.MAX_VALUE).random())
```

Comment:
Here, `Int.MIN_VALUE = -2147483648` and `Int.MAX_VALUE = 2147483647`. However, the RGB colors are between 0 and FFFFFF càd 0 -> 16777215. I suggest changing this to : 
`import java.awt.Color.BLACK `
`import java.awt.Color.WHITE `
`... `
`    it[color] = Color((BLACK.rgb..WHITE.rgb).random())`